lib/openstudio/lib/openstudio_aws_wrapper.rb

```diff
@@ -291,8 +336,15 @@ def save_private_key(filename)
 
   # save off the worker public/private keys that were created
   def save_worker_keys(directory = '.')
-    File.open("#{directory}/ec2_worker_key.pem", 'w') { |f| f << @worker_keys.private_key }
-    File.open("#{directory}/ec2_worker_key.pub", 'w') { |f| f << @worker_keys.public_key }
+    wk = "#{directory}/ec2_worker_key.pem"
+    logger.info "Saving worker private key in #{wk}"
+    File.open(wk, 'w') { |f| f << @worker_keys.private_key }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.